.vsts-ci.yml

```diff
@@ -15,134 +15,141 @@ pr:
+        - task: PublishBuildArtifacts@1
+          displayName: Publish Artifact Symbols
+          condition: and(eq(variables['system.pullrequest.isfork'], false), eq(variables['_BuildConfig'], 'Release'))
+          inputs:
+            PathtoPublish: '$(Build.SourcesDirectory)\artifacts\SymStore\$(_BuildConfig)'
+            ArtifactName: 'NativeSymbols'
 
-    - job: Ubuntu_16_04
-      displayName: 'Ubuntu 16.04'
-      pool:
-        name: Hosted Ubuntu 1604
-      strategy:
-        matrix:
-          ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
-            Debug:
-              _BuildConfig: Debug
+      - job: Ubuntu_16_04
+        displayName: 'Ubuntu 16.04'
+        pool:
+          name: Hosted Ubuntu 1604
+        strategy:
+          matrix:
+            ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
+              Debug:
+                _BuildConfig: Debug
+                _SignType: none
+                _DotNetPublishToBlobFeed : false
+            Release:
+              _BuildConfig: Release
               _SignType: none
               _DotNetPublishToBlobFeed : false
-          Release:
-            _BuildConfig: Release
-            _SignType: none
-            _DotNetPublishToBlobFeed : false
-      steps:
-      - checkout: self
-        clean: true
-      - script: eng/common/cibuild.sh
-          --configuration $(_BuildConfig)
-          --prepareMachine
-        displayName: Build
+        steps:
+        - checkout: self
+          clean: true
+        - script: eng/common/cibuild.sh
+            --configuration $(_BuildConfig)
+            --prepareMachine
+          displayName: Build
 
-    - job: OSX_10_13
-      displayName: 'OSX'
-      pool:
-        name: Hosted macOS
-        vmImage: macOS-10.13
-      strategy:
-        matrix:
-          ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
-            Debug:
-              _BuildConfig: Debug
+      - job: OSX_10_13
+        displayName: 'OSX'
+        pool:
+          name: Hosted macOS
+          vmImage: macOS-10.13
+        strategy:
+          matrix:
+            ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
+              Debug:
+                _BuildConfig: Debug
+                _SignType: none
+                _DotNetPublishToBlobFeed : false
+            Release:
+              _BuildConfig: Release
               _SignType: none
               _DotNetPublishToBlobFeed : false
-          Release:
-            _BuildConfig: Release
-            _SignType: none
-            _DotNetPublishToBlobFeed : false
-      steps:
-      - checkout: self
-        clean: true
-      - script: eng/common/cibuild.sh
-          --configuration $(_BuildConfig)
-          --prepareMachine
-        displayName: Build
+        steps:
+        - checkout: self
+          clean: true
+        - script: eng/common/cibuild.sh
+            --configuration $(_BuildConfig)
+            --prepareMachine
+          displayName: Build
```

Comment:
You need to include the `post-build.yml` template at the end of the build definition. See example here: https://github.com/dotnet/arcade/pull/4556/files#diff-c88223c5b65116dd5625673d42760adfR140